submissions/markcampanelli/pr_config.json

```diff
@@ -0,0 +1,6 @@
+{
+    "RUN_SCORER": true,
+    "REQUIREMENTS": "./requirements.txt",
+    "SUBMISSION_MAIN": "./create benchmark_data.py",
```

Comment:
```suggestion
    "SUBMISSION_MAIN": "./create_benchmark_data.py",
```

Reply:
Maybe correcting this file name will fix it, I haven't wrapped my head around what you are doing with pytest here.

Reply:
Thanks for catching my typo. I have fixed it, but I still only see the documentation part of the build kicking off.

In the `pvfit` repo, I have a test module that has one unit test that creates the benchmark input files and another that runs the benchmark. So the script here uses pytest to run the one unit test that creates the benchmark input files.